.github/workflows/enforce-labels.yml

```
@@ -0,0 +1,13 @@
+---
+name: Enforce PR labels
+
+on:
+  pull_request:
+    types: [labeled, unlabeled, opened, reopened, synchronize]
+jobs:
+  enforce-labels:
+    name: Enforce labels
+    runs-on: ubuntu-latest
```

```suggestion
    runs-on: ubuntu-22.04
```
❓  Should this be pinned?

---

It should. Thanks!